doc/ch-fromhost_desc.rst

```diff
@@ -246,7 +246,25 @@ Inject the Cray-enabled MPI libraries into the image, and then run
 :code:`ldconfig`::
 
   $ ch-fromhost --cray-mpi /var/tmp/baz
-
+  asking ldconfig for shared library destination
+  /sbin/ldconfig: Can't stat /libx32: No such file or directory
+  /sbin/ldconfig: Can't stat /usr/libx32: No such file or directory
```

Comment:
Could you add a note stating that these errors are non-fatal and link to issue #732 ?